src/selectors/petitionMetrics.js

```diff
@@ -13,6 +13,7 @@ export default (petition) => {
   const amountVotes = petition.supporters.amount;
   const timePercentage = getTimePercentage(daysRemaining, daysToVote);
   const votesPercentage = calculatePercentage(amountVotes, requiredVotes);
+  const votingActive = requiredVotes > 0;
```

Comment:
I left this intentionally to `> 0` as the API does not provide a `-1` value yet. Should be changed later